src/components/InviteMissing.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import { Link, Redirect } from 'react-router-dom';
+
+import Loading from './Loading';
+import { getAuthMemberDocIsLoaded, getAuthMemberDoc } from '../connectors';
+
+// TODO redirect if you are registered
+function InviteMissing({ authFirebaseUser, authIsLoaded, authMemberDocIsLoaded, authMemberDoc }) {
```

Comment:
I would love a doc comment explaining what the `invite_missing` page is supposed to refer to. I don't find it obvious from the name nor it's behavior really, and it deviates from a more familiar RESTful url structure.